src/flag_gems/ops/index_put.py

```diff
@@ -54,12 +54,12 @@ def generate_index_put_kernel(
     code.writeline("@libentry()")
     code.writeline("@libtuner(")
```

Comment:
No need libtuner. We can use default value in kernel function for BLOCK_SIZE0 and BLOCK_SIZE1